src/js/views/patients/worklist/filters_views.js

```diff
@@ -64,6 +69,83 @@ const ClinicianDropList = Droplist.extend({
   },
 });
 
+const DateFilterView = View.extend({
+  template: hbs`
+    <button class="button-filter js-date">
```

Comment:
I'm a little on the fence about this.. typically when there's a single child tag this'd be better described as `tagName: 'button', className: 'button-filter', triggers: { 'click': 'click' }`

In a component `ViewClass` can be a function that uses the state to determine which view to pass.. _so_ we could remove the need for template logic and templateContext as well... there'd be a little bit of duplication.. particularly when the "Added:" label changes..  so I'm not 100% sold on that.

Reply:
I like the ViewClass approach. But once the value of the date type can be change, we may still need to rely on the template context. I think that could still be a fairly clean solution, though.